stingray/tests/test_lightcurve.py

```diff
@@ -800,6 +800,30 @@ def test_sort_reverse(self):
         new_lc_long = lc_long[:]  # Copying into a new object
         assert new_lc_long.n == lc_long.n
 
+    def test_to_lightkurve(self):
```

Comment:
To test this properly, we need to add the `lightkurve` dependency to one of the test cases in Travis (see file `.travis.yml`), define some global variable `HAS_LIGHTKURVE` that is `True` if `lightkurve` is installed, and decorate this test function with something like
```
@pytest.mark.skipif('not HAS_LIGHTKURVE')
def test_to_lightkurve(self):
(...)

```
We do this for a number of tests, you can take example from there.